src/libs/PusherConnectionManager.js

```diff
@@ -33,6 +33,7 @@ function init() {
                 socket_id: socketID,
                 channel_name: channel.name,
                 doNotRetry: true,
+                forceNetworkRequest: true,
```

Comment:
Trying to understand how this works - it looks like this param gets passed [here](https://github.com/Expensify/Expensify.cash/blob/e2369bc2a84fffd549fe2b099626db4cf872dd36/src/libs/Pusher/pusher.js#L51-L61), and are there required changes to the `Push_API`?

Reply:
Hmm no it does not get passed there. The params are sent with the request to the API via the custom authorizer here

https://github.com/Expensify/Expensify.cash/blob/e2369bc2a84fffd549fe2b099626db4cf872dd36/src/libs/Pusher/pusher.js#L47-L49

Get pushed into the `networkRequestQueue` here
https://github.com/Expensify/Expensify.cash/blob/e2369bc2a84fffd549fe2b099626db4cf872dd36/src/libs/Network.js#L163-L177

And then processed here
https://github.com/Expensify/Expensify.cash/blob/e2369bc2a84fffd549fe2b099626db4cf872dd36/src/libs/Network.js#L113-L121

Where we determine if they can be made here
https://github.com/Expensify/Expensify.cash/blob/e2369bc2a84fffd549fe2b099626db4cf872dd36/src/libs/Network.js#L41-L45

The param is not used by the auth endpoint in Web-E it's just a param used in the front end to bypass the request queue.

Reply:
OHHH, gotcha makes sense. Didn't make the connection that this was just a `Network.post` parameter. Thank you for the explanation.